extensions/opentelemetry-resteasy-reactive/runtime/src/main/java/io/quarkus/opentelemetry/resteasy/reactive/runtime/OpenTelemetryReactiveServerFilter.java

```diff
@@ -9,13 +7,10 @@
 import io.opentelemetry.instrumentation.api.instrumenter.LocalRootSpan;
 import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
 
-/**
- * Handles RESTEasy Reactive (via Vert.x)
- */
 public class OpenTelemetryReactiveServerFilter {
 
     @ServerRequestFilter
-    public void filter(SimpleResourceInfo resourceInfo) throws IOException {
+    public void filter(SimpleResourceInfo resourceInfo) {
         Span localRootSpan = LocalRootSpan.current();
```

Comment:
Why are you using LocalRootSpan, instead of span? Not sure if the current span will never have a local parent.

Reply:
I used what already have in https://github.com/quarkusio/quarkus/blob/3.2.0.Final/extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/intrumentation/resteasy/OpenTelemetryReactiveServerFilter.java#L19.

If that is incorrect, I propose we add a new commit that fixes both (existing and new) cases. Feel free to push the proper fix into this PR as an extra commit

Reply:
I don't think it will cause problems because these will track inbound connections. They should have, in almost all cases, an empty local parent.
I was just curious about the option.